### ISSUE TITLE
❓ [Question] How to convert Torch-TensorRT module to TRT engine?

### ISSUE THREAD

**Issue by lingffff** [pytorch/TensorRT#1070]
## ❓ Question

How to convert Torch-TensorRT module (*.ts) to TRT engine? Is there any Python API to do that?
## What you have already tried

In examples, I found
```cpp
auto engine = torch_tensorrt::ts::convert_method_to_trt_engine(mod, "forward", compile_spec);
```
in https://github.com/pytorch/TensorRT/blob/master/examples/int8/qat/main.cpp

If this is the correct way to do converting? If yes, is there any Python API?
## Environment

amd64 + Linux
all software is newest version

**Comment by narendasan** [comment]
Yes this is the way, there is a python api by the same name in `torch_tensorrt.ts`
